docker/docker-compose.yml

```diff
@@ -13,8 +13,19 @@ services:
     ports:
       - "9000:9000"
 
+  redis:
+    image: redis:6.0.2 # Stable redis
+    hostname: redis
+    volumes:
+      - redis-data:/data
+    networks:
+      - xain-fl-rs
```

Comment:
i noticed that we still have the `-rs` rust suffix in our docker files and should probably rename that without using a suffix at one point (maybe not now).

Reply:
Fixed 👍